src/imgui/ImGuiDebugger.cc

```diff
@@ -257,7 +257,7 @@ void ImGuiDebugger::showMenu(MSXMotherBoard* motherBoard)
 						im::ID(id, [&]{
 							if (auto displayName = std::string(FileOperations::stripExtension(FileOperations::getFilename(name)));
 							    ImGui::Selectable(displayName.c_str())) {
-								selectedFile = name;
+								selectedFile = displayName;
```

Comment:
May I suggest to do this stem-calculation already in `getBreakpointFiles()`.
The small advantage is that then here we don't (again) need to convert the stem from `std::string_view` to `std::string` (`getBreakPointFiles()` already needs to do that to be able to store the string).